packages/react-native-codegen/src/parsers/typescript/modules/index.js

```diff
@@ -243,6 +243,27 @@ function translateTypeAnnotation(
       }
     }
     case 'TSTypeLiteral': {
+      // if there is TSIndexSignature, then it is a dictionary
+      if (typeAnnotation.members) {
+        const indexSignatures = typeAnnotation.members.filter(
+          member => member.type === 'TSIndexSignature',
+        );
+        if (indexSignatures.length > 0) {
+          // check the property type to prevent developers from using unsupported types
+          const propertyType = indexSignatures[0].typeAnnotation;
+          translateTypeAnnotation(
```

Comment:
We are not using the returned value of `translateTypeAnnotation`. Is it right?

Reply:
The return value is ignored, since we don't really have `DictionaryTypeAnnotation`, there is no place to return it. But I call this function to check if the type is supported.